src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The name of the interface `AreaCalculator` is not following the checklist item which advises against using verbs for class/interface names. Consider renaming it to something like `AreaComputable` or `AreaProvider` to better convey that it's an interface providing an area calculation capability.